public/app/core/components/Upgrade/UpgradeBox.tsx

```diff
@@ -88,11 +79,150 @@ const getUpgradeBoxStyles = (theme: GrafanaTheme2, size: ComponentSize) => {
         outline: 2px solid ${theme.colors.primary.main};
       }
     `,
-    buttonSecondary: css`
-      color: ${theme.colors.text.secondary};
-    `,
     icon: css`
       margin: ${theme.spacing(0.5, 1, 0.5, 0.5)};
     `,
   };
 };
+
+export interface UpgradeContentProps {
+  image: string;
+  featureUrl?: string;
+  featureName: string;
+  description?: string;
+  listItems: string[];
+  caption?: string;
+}
+
+export const UpgradeContent = ({
+  listItems,
+  image,
+  featureUrl,
+  featureName,
+  description,
+  caption,
+}: UpgradeContentProps) => {
+  const styles = useStyles2(getUpgradeContentStyles);
+  return (
+    <div className={styles.container}>
+      <div className={styles.content}>
+        <h3 className={styles.title}>Get started with {featureName}</h3>
+        {description && <h6 className={styles.description}>{description}</h6>}
+        <ul className={styles.list}>
+          {listItems.map((item, index) => (
+            <li key={index}>
+              <Icon name={'check'} size={'xl'} className={styles.icon} /> {item}
+            </li>
+          ))}
+        </ul>
+        {featureUrl && (
+          <LinkButton fill={'text'} href={featureUrl} className={styles.link} target="_blank" rel="noreferrer noopener">
```

Comment:
Actually, this doesn't work to open the link in a new tab and I don't know why.

Reply:
There's another component for vertical layout and the link there didn't have `target="_blank"` attribute.